backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -265,11 +265,10 @@ private async void OnIsarMissionUpdate(object? sender, MqttReceivedArgs mqttArgs
                 return;
             }
 
-            if (flotillaMissionRun.Status == status) { return; }
-
             if (flotillaMissionRun.IsLocalizationMission())
             {
-                if (status == MissionStatus.Successful || status == MissionStatus.PartiallySuccessful)
+                var task = flotillaMissionRun.Tasks.FirstOrDefault();
+                if (task != null && (task.Status == Database.Models.TaskStatus.Successful || task.Status == Database.Models.TaskStatus.PartiallySuccessful))
```

Comment:
This probably works fine, but it could be a good idea to just check if any tasks have succeeded, in case the ordering is not consistent.

Reply:
I guess the first task could fail for some weird reason and then we move on to the second tag and if that one works then the localization worked, can change